NuKeeper.Inspection/Files/FolderFactory.cs

```diff
@@ -31,19 +32,20 @@ private static string NuKeeperTempFilesPath()
         private static string GetUniqueTemporaryPath()
         {
             var uniqueName = Guid.NewGuid().ToString("N", CultureInfo.InvariantCulture);
-            return Path.Combine(NuKeeperTempFilesPath(), uniqueName);
+            return Path.Combine(NuKeeperTempFilesPath(), $"{FolderPrefix}{uniqueName}");
         }
 
         /// <summary>
         /// Cleanup folders that are not automatically have been cleaned.
-        /// Only delete folders older than 1 hour to 
+        /// Only delete folders older than 1 hour
+        /// Only delete folders that contain repositories
         /// </summary>
         public void DeleteExistingTempDirs()
         {
             var dirInfo = new DirectoryInfo(NuKeeperTempFilesPath());
             var dirs = dirInfo.Exists ? dirInfo.EnumerateDirectories() : Enumerable.Empty<DirectoryInfo>();
             var filterDatetime = DateTime.Now.AddHours(-1);
-            foreach (var dir in dirs.Where(d => d.LastWriteTime < filterDatetime))
+            foreach (var dir in dirs.Where(d => d.Name.StartsWith(FolderPrefix, StringComparison.InvariantCultureIgnoreCase) && d.LastWriteTime < filterDatetime))
```

Comment:
Maybe we can do this filtering in a seperate private function. Giving it a nice name could help others understanding what is happening.

Reply:
The idea is a good one, and it looks like the code will work, but I agree with extracting a named function, maybe something like ` foreach (var dir in dirs.Where(d => IsExpiredNuKeeperRepoFolder(d))`

Reply:
> Maybe we can do this filtering in a seperate private function. Giving it a nice name could help others understand what is happening.

Am working on that as well as testing that said function in a unit-test